src/styles/app/46-quick-note.css

```
@@ -0,0 +1,121 @@
+	resize: none;
+	border: 0;
+	outline: none;
+	background: transparent;
+	color: color-mix(in srgb, var(--text-primary) 96%, var(--text-secondary));
+	padding: 30px 28px 72px;
+	font: inherit;
+	font-size: var(--editor-inline-font-size);
+	line-height: 1.68;
+	letter-spacing: 0;
+	caret-color: var(--interactive-accent);
+	text-rendering: optimizeLegibility;
```

Stylelint (17.9.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)